daily-challenges/2024-09/17/khanh5923_17.spec.js

```diff
@@ -0,0 +1,49 @@
+function isPrime(input){
+    if(input == 1) return console.log(`${input} khong phai la so nguyen to`)
```

Comment:
Luôn sử dụng ngoặc với các câu điều kiện, dù chỉ có 1 mệnh đề.
```suggestion
    if(input == 1) {
        return console.log(`${input} khong phai la so nguyen to`)
    }
```